### PR TITLE
Extracting BNE edition

### PR DESCRIPTION
**Pull request by ChefKeeper**
Greatly improved BattleNet edition extraction, few sounds are missing though but I don't know why.

### CODE REVIEW

**Comment by ChefKeeper** [comment]
Bump :)
Tell me if you have questions, or if I did something bad I should improve

**Comment by timfel** [comment]
So, I can't test this. Will it work now even if the tome.* files are missing?

File: wartool.h [hunk 1]
```diff
@@ -35,9 +35,9 @@
 --  General
 ----------------------------------------------------------------------------*/
 
-#define VERSION "2.4.2" // Version of extractor wartool
+#define VERSION "2.4.1" // Version of extractor wartool
```
Comment (timfel) [inline wartool.h]: You should not downgrade the version.

File: wartool.h [hunk 2]
```diff
@@ -35,9 +35,9 @@
...
 
-const char NameLine[] = "wartool V" VERSION " for Stratagus, (c) 1998-2017 by The Stratagus Project.\n"\
+const char NameLine[] = "wartool V" VERSION " for Stratagus, (c) 1998-2016 by The Stratagus Project.\n"\
```
Comment (timfel) [inline wartool.h]: Please keep the copyright current.

**Comment by ChefKeeper** [comment]
Hey yes it's supposed to work without the tome.* files for BNE (they are in the mpq itself)

**Comment by ChefKeeper** [comment]
You should just test if it still works for non BNE, but it should as I only changed the path files of the BNE

**Comment by timfel** [comment]
Thanks a lot!

**Pull request merged**